tasks/sep/unichain-003-petra-l1-upgrade-new-prestate/.env

```diff
@@ -0,0 +1,19 @@
+ETH_RPC_URL="https://ethereum-sepolia.publicnode.com"
+SAFE_NONCE="25"
```

Comment:
This is deprecated if you see the logs of the sim-sequence in the CI. 
you need to replace to 
SAFE_NONCE_XXXX= where XXX is the PAO with unichain looks like is `0xd363339eE47775888Df411A163c586a8BdEA9dbf`